packages/teams-js/src/public/meeting.ts

```diff
@@ -518,4 +518,25 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.meetingReactionReceived', handler);
   }
+  /**
+   * nested namespace for Meeting Service
+   *
+   * @beta
+   */
+  export namespace meetingAppShareButton {
+    /**
+     * Allows an app to hide/show app share button in a meeting
+     * @param isGlobalAppShareButtonVisible is a boolean flag to set show or hide app share button
+     *
+     * @beta
+     */
+    export function setAppShareButtonVisibility(isGlobalAppShareButtonVisible: boolean): Promise<void> {
```

Comment:
global should also be removed from the name of the parameter you're passing to this function